### ISSUE TITLE
training accuracy for first few epochs stalled at 0.5

### ISSUE THREAD

**Issue by danbaterisna** [danbaterisna/ethical-midget-in-a-box#1]
to quote sir mac:

"Training accuracy at 0.5 is a chance level accuracy (like flipping a coin when deciding whether fake or real), thus, the system is not learning anything, either the learning rate is very low or equal to zero. You have to check the hyper parameters. Verify this:

For the last few layers: learning rate, learning rate multiplier, and other hyper parameters related to learning should be non-zero. For the first few layers: these hyper parameters should be very low or zero. This is because in fine-tuning, we only adjust the last few layers.

If these hyper parameters ok and the training accuracy is still at chance level, try to:

(1) Balance the number of samples for each category. Make sure that for each iteration, you randomly sample (with equal probability) from the mixture of the fake and real samples.
(2) For every 3 epochs, decrease the learning rate by a factor of 10. For example, if the learning rate is initially 0.01, after three epochs, the learning rate will be 0.001. Then after 6 epoch, the learning rate will be 0.0001, etc...
(3) If still no improvement, modify the momentum (any value between 0.90 to 0.99). The momentum is the numerical value that you multiply to the present weight values w(t) to determine the weight values for the next iteration w(t+1). That is: w(t+t) = momentum*w(t) + learning_rate*loss_function. 

Hint: if you have a learning system, the initial loss or accuracy will be worse for the the first few iterations then it will slowly improve. Let me know the results."

**Comment by danbaterisna** [comment]
644e91d has much better results
